components/x11/xf86-video-ati/Makefile

```diff
@@ -33,12 +34,14 @@ CONFIGURE_DEFAULT_DIRS = no
 include $(WS_TOP)/make-rules/configure.mk
 include $(WS_TOP)/make-rules/ips.mk
 
-COMPONENT_PREP_ACTION =        ( cd $(@D) && \
-                                        aclocal -I. && \
-                                        autoheader && \
-                                        libtoolize --automake --copy --force && \
-                                        automake -a -f -c &&\
-                                        autoconf )
+COMPONENT_PREP_ACTION = \
+ ( cd $(@D) && \
+    aclocal -I. && \
+    autoheader && \
+    libtoolize --automake --copy --force && \
+    automake -a -f -c && \
+    autoconf && \
+    cp $(COMPONENT_DIR)/srcs/i2c/*.h $(COMPONENT_DIR)/srcs/i2c/*.c src/ )
```

Comment:
Why doing this cp? Perhaps, better to place files initially as needed? Why modifying files in src, but not $(BUILD_DIR)?